client/redux/orders.js

```diff
@@ -42,13 +66,32 @@ export const fetchSingleCart = (id) => {
   };
 };
 
-export const addItem = (item) => {
+export const addItem = (productId, orderId, qty) => {
   // note: item {orderId:, productId:, quantity:}
-  console.log('in item thunk', item);
   return async (dispatch) => {
     try {
+      const item = { productId, orderId, quantity: qty };
```

Comment:
Don't forget to delete the quantity key

Reply:
this works as is--I would also have to change the qty arg of this thunk to quantity for your comment to work--is that what you would prefer?

Reply:
Oh no, if it works nevermind! Disregard my comment :)